core/indigo-core/reaction/reaction_json_loader.h

```diff
@@ -46,6 +47,7 @@ namespace indigo
     class QueryMolecule;
     class MoleculeJsonLoader;
 
+    using namespace rapidjson;
```

Comment:
Please avoid `using namespace ..` in header files on the global level.

Reply:
Fixed